app/src/main/java/com/ferranpons/issposition/MainActivity.kt

```diff
@@ -0,0 +1,292 @@
+package com.ferranpons.issposition
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.content.pm.PackageManager
+import android.graphics.Bitmap
+import android.graphics.Matrix
+import android.graphics.drawable.BitmapDrawable
+import android.location.Criteria
+import android.location.Location
+import android.location.LocationListener
+import android.location.LocationManager
+import android.net.ConnectivityManager
+import android.os.Bundle
+import android.support.v4.app.ActivityCompat
+import android.support.v7.app.AppCompatActivity
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import android.widget.*
+import butterknife.BindView
+import butterknife.ButterKnife
+import butterknife.OnClick
+import com.ferranpons.issposition.issTracking.*
+import com.ferranpons.issposition.passTimes.PassTimesAdapter
+import com.ferranpons.issposition.peopleInSpace.PeopleAdapter
+import com.google.android.gms.common.ConnectionResult
+import com.google.android.gms.common.GooglePlayServicesUtil
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.android.gms.maps.model.CameraPosition
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+
+class MainActivity : AppCompatActivity(), IssTrackingViewInterface {
+
+    private val map: GoogleMap? = null
+    private var issTrackingPresenter: IssTrackingPresenterInterface? = null
```

Comment:
Usually people use a Lazy initialize so you don't need to accept nulls for that field.